test/LanguageServer.Engine.Tests/TestProjects/RedefineProperty.SameFile.csproj

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Property1>true</Property1>
+    <Property2>false</Property2>
+    <Property2 Condition=" '$(Property1)' == 'true' ">true</Property2>
+  </PropertyGroup>
+</Project>
```

Comment:
In the long term it will probably be better if we can operate test documents fully in memory and avoid storing them on disk, including as content, copying to output folder and so on. Feels more convinient and reliable. But that's the long term, fully ok having them in this state for now

Reply:
We might be able to store them as embedded resources but I think MSBuild wants physical files to work with (at some point in the process anyway)…

Reply:
It’s been a while, but from memory you’re always better off with a temp directory, even though you can construct a project in-memory (because of the way SDK props/targets are resolved and up-to-date checks are performed)…